test/integrationTests/hoverProvider.integration.test.ts

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
Missing copyright header

Reply:
@akshita31 if you install the `tslint` extension for vs code you will get notification for this and similar issues.